### PR TITLE
speedtest: update page

### PR DESCRIPTION
**Pull request by Abhishek-1804**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/speedtest.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # speedtest
 
 > Official command-line interface for testing internet bandwidth using <https://speedtest.net>.
-> Note: some platforms link `speedtest` to `speedtest-cli`. If some of the examples in this page don't work, see `speedtest-cli`.
+> Note: some platforms link `speedtest` to `speedtest-cli` or other tools like `librespeed`, which can also be installed as `speedtest` on certain linux distributions.
```
Comment (spageektti) [inline pages/common/speedtest.md:4]:
```suggestion
> Note: some platforms link `speedtest` to `speedtest-cli` or other tools like `librespeed`, which can also be installed as `speedtest` on certain Linux distributions.
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. I wish a wonderful christmas time!

**Comment by Managor** [comment]
I am blind. I dodn't notice that explanation when I looked at the page last time. That's the reason why I thought the commands were broken

File: pages/common/speedtest.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 > More information: <https://www.speedtest.net/apps/cli>.
```
Comment (Managor) [inline pages/common/speedtest.md:5]:
```suggestion
> These command examples apply only to the official client.
> More information: <https://www.speedtest.net/apps/cli>.
```

**Review by Managor — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by Managor — APPROVED** [review]

**Pull request merged**